app/src/main/java/org/hyperledger/besu/controller/IbftLegacyBesuControllerBuilder.java

```diff
@@ -59,7 +59,7 @@ protected MiningCoordinator createMiningCoordinator(
       final MiningConfiguration miningConfiguration,
       final SyncState syncState,
       final EthProtocolManager ethProtocolManager) {
-    return new NoopMiningCoordinator(miningConfiguration);
+    return new NoopMiningCoordinator();
```

Comment:
This might affect the gas price reported in EthStats for ibft1 and possibly the upper and lower bounds in EthFeeHistory. But ibft1 is also deprecated and shouldn't be used other than to migrate so maybe that's ok to affect these APIs?

Reply:
I think it's ok to have those side effects for a deprecated mode. I couldn't find IBFT legacy in the docs at all, anywhere else we would record this or need to update? Worth adding a comment to IbftLegacyBesuControllerBuilder?